### DISCUSSION TITLE
aggregateRaw with Regular Expression inside $in

### DISCUSSION THREAD

**Discussion by vinayprolitus** [prisma/prisma#21135] [Q&A]
### Question

I was trying to search for case insensitive value using $in
Query is something like this
```js
{ gender: { '$in': [ /M/i, /F/i ] } }
```
I created this object with 
```js
new RegExp(value, "i")
```
But looks like this isn't working (no error thrown) as it is not returning any values where it should have. Although its working when I am using it like 
```js
{ gender: { '$in': [ "M", "F" ] } } // but I want this to be case insensitive.
```
MongoDB has this 
https://www.mongodb.com/docs/manual/reference/operator/query/regex/#-regex-vs.--pattern--syntax

### How to reproduce (optional)

Just try to use regular expression pattern with $in

### Expected behavior (optional)

Should have returned values as I have confirmed there are values

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @vinayprolitus 👋 

Thank you for raising this question.

I can confirm this behavior as well when using `aggregateRaw` command. 

This expression `{ gender: { $in: [ /M/i, /F/i ] } }` on MongoDB atlas returns the result set but with Prisma `aggregateRaw` command, it returns an empty array [].

Can you please open a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml)  so that our engineers can take a look?

**Reply by vinaykharayat** [reply]
Done

**Reply by ludralph** [reply]
Link to the created bug [issue](https://github.com/prisma/prisma/issues/21150)
